src/main/java/org/snomed/snowstorm/core/data/services/ConceptUpdateHelper.java

```diff
@@ -181,7 +180,6 @@ PersistedComponents saveNewOrUpdatedConcepts(
 				}
 
```

Comment:
The main protagonist for this PR is `ConceptUpdateHelper` which has been tweaked to correctly restore the Concept's details.